wrappers/python/simtk/openmm/app/internal/pdbstructure.py

```diff
@@ -795,11 +798,14 @@ def __init__(self, pdb_line, pdbstructure=None):
         try: self.formal_charge = int(pdb_line[78:80])
         except ValueError: self.formal_charge = None
         # figure out atom element
-        try:
-            # Try to find a sensible element symbol from columns 76-77
-            self.element = element.get_by_symbol(self.element_symbol)
-        except KeyError:
-            self.element = None
+        if self.element_symbol == extraParticleIdentifier:
+            self.element = 'EP'
```

Comment:
The way I did it allows for `PDBStructure` to separately return 'extra particle' and 'atom for which element could not be determined by `self.element = element.get_by_symbol(self.element_symbol)`, and these are passed respectively as `element = 'EP'` and `element = None`. 

`PDBFile` then picks this up and applies some extra rules to try to work out elements: https://github.com/pandegroup/openmm/blob/master/wrappers/python/simtk/openmm/app/pdbfile.py#L119

For doing what you're proposing, you'd need to change the meaning of `None` at `PDBFile` from "unknown element - do some more working out" to "extra particle", so we can't apply these extra rules in `PDBFile` and we're getting element `None` returned in topology by `PDBFile` where elements used to be worked out by this code. 

On a practical note for what this EP code is meant to add: think e.g. loading a PDB file that has both 'EP' elements and some blank / all other blank element fields - user marks EPs but hopes the other atom element's will be determined by code. 

We can add `EP' as an `Element` object if not comfortable with this being a string in `PDBStructure`?